docs/graphql/manual/api-reference/schema-metadata-api/remote-relationships.rst

```diff
@@ -0,0 +1,228 @@
+.. meta::
+   :description: Manage remote relationships with the Hasura schema/metadata API
+   :keywords: hasura, docs, schema/metadata API, API reference, remote joins, remote relationships
+
+Schema/Metadata API Reference: Remote Relationships
+===================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Remote Relationships will ensure independent remote schemas and related data types can be joined natively via Hasura.
+
+.. _create_remote_relationship:
+
+create_remote_relationship
+--------------------------
+
+``create_remote_relationship`` is used to create a new remote relationship with an existing remote schema.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+  {
+    "type": "create_remote_relationship",
+    "args": {
+      "name": "sample_remote_relationship",
+      "table": "users",
+      "hasura_fields": [
+        "id"
+      ],
+      "remote_schema": "my-remote-schema",
+      "remote_field": {
+        "messages": {
+          "arguments": {
+            "where": {
+              "id": {
+                "eq": "$id"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+
+.. _create_remote_relationship_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - name
+     - true
+     - RemoteRelationshipName_
+     - Name of the remote relationship
+   * - table
+     - true
+     - :ref:`QualifiedTable <QualifiedTable>`
+     - Object with table name and schema
+   * - hasura_fields
+     - true
+     - [:ref:`PGColumn <PGColumn>`]
+     - List of PG columns in the table that will be used in the ``remote_field`` object.
```

Comment:
```suggestion
     - Column(s) in the table that is used for joining with remote schema field. All join keys in ``remote_field`` must appear here.
```

I thought `hasura_fields` was removed as it was redundant. Should check.

Reply:
The `hasura_fields` are being used, I verified now.